deploy/k8s-app-deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: "apps/v1"
+kind: "Deployment"
+metadata:
+  name: "link-app"
+  labels:
+    app: "link-app"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: "link-app"
+  strategy: {}
+  template:
+    metadata:
+      labels:
+        app: "link-app"
+    spec:
+      containers:
+      - name: "link-app"
+        image: path.to.docker/image/link-app-docker:latest
```

Comment:
I take it this is piece still needs to be worked out?

Reply:
Yes, that's right.